pkg/render/node.go

```diff
@@ -642,6 +647,8 @@ func (c *nodeComponent) createCalicoPluginConfig() map[string]interface{} {
 		"policy": map[string]interface{}{
 			"type": "k8s",
 		},
+		"policy_setup_timeout_seconds": policySetupTimeoutSeconds,
+		"endpoint_status_dir":          filepath.Join(c.varRunCalicoVolume().VolumeSource.HostPath.Path, "status"),
```

Comment:
`/var/run/calico/status` is a pretty generic path. Can we make this more intuitive?

e.g., 

`/var/run/calico/policy_program_status` ?

Reply:
Would be nice to align with the field name too - we seem to be using `endpoint_status` and `endpointStatus` elsewhere, so maybe this should be `/var/run/calico/endpoint_status`

Reply:
Happy to take this onboard, Shaun also suggested:

    I might use a - instead of _ in a dir name (unless there was precedent in an adjacent dir

If that works for you?

Reply:
:shrug: should be fine either way, main thing is the name. I'd double check the precedent thing though - what other multi-part dir names do we create?

Reply:
Had a quick skim of some reference docs and occurrences of `Mkdir` in our repo and most I saw seemed to use dashes, so I'll go with that.